R/utils.R

```diff
@@ -2,3 +2,15 @@
 # url builder (take base url, asset type and identifier and output format?)
 # if we write the functions outside the classes and set them later in the class
 # code readability is higher imo
+
+#' @description
+#' Converts R list into JSON-like string.
+#'
+#' @keywords internal
```

Comment:
Is there are particular reason to exclude this file from the documentation index?
Also,  one could add a `@title` and `@return` tag

p.s. I didn't know about that tag, pretty cool!

Reply:
Because it's an auxiliary function? from [here](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html): "`@keywords internal` is useful because it removes the function from the documentation index; it’s useful for functions aimed primarily at other developers, not typical users of the package."

Reply:
I added `#' @return JSON-like string` , do you think it's enough?

Reply:
Yes, sounds good! thx